relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -312,6 +312,12 @@ fn extract_transaction_metrics_inner(
         None => return,
     };
 
+    let mut push_metric_if_ok =
+        |potential_metric: Result<Metric, MetricInitError>| match potential_metric {
+            Ok(metric) => push_metric(metric),
+            Err(error) => relay_log::error!("{}", error),
+        };
```

Comment:
This piece of code is almost identical to the one in `sessions.rs`. I thought about a way to not duplicate it, but all alternatives that I could think of are less readable and maintainable. I'm open to suggestions.